tests/unit_tests/libs_tests/ECS_tests/Systems_tests/Decrease_LifeTime_System_tests.cpp

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Maybe you could had some unit / functional testing of multiple system combination (LifeTime + Death for example)